models/kyberswap/ethereum/kyberswap_ethereum_schema.yml

```diff
@@ -0,0 +1,97 @@
+version: 2
+
+models:
+  - name: kyberswap_ethereum_trades
+    meta:
+      blockchain: ethereum
+      sector: dex
+      project: kyberswap
+      contributors: 
+    config:
+      tags: ['ethereum','dex','trades', 'kyberswap', 'zhongyiio', 'hosuke', 'msilb7']
+    description: >
+        Kyberswap trades on Ethereum across all contracts and versions. This table will load dex trades downstream.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+      - check_dex_seed:
+          blockchain: ethereum
+          project: kyberswap
+          version: dmm
```

Comment:
Hi @gregshestakovlido for `check_dex_seed` to work, we need to add the corresponding seed records in this file: https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv

Reply:
added